### PR TITLE
HPCC-20218 Update Data Handling doc and images

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

@JamesDeFabia please review 

http://10.240.32.243/view/Docs%207.0/job/DocDev70/49/CE=ubuntu-14.04-amd64-1/artifact/build-docs/Release/docs/EN_US/HPCCDataHandling_EN_US-7.0.0-documentation0.pdf#page=7

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [X] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [X] The commit message is properly formatted and free of typos.
  - [X] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [X] My change requires a change to the documentation.
  - [X] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [X] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [X] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:

Successful Unit Testing: http://10.240.32.243/view/Docs%207.0/job/DocDev70/

### CODE REVIEW

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
I know the intent is to make the instructions software-agnostic, but there are some missing details. We need to tell users to choose SFTP as the protocol. Hostname, port, user credentials... See the steps in the Jira. 
We do not want to document the 3rd party software, but we need to inform users what info they need to know to use a tool to secure copy over SFTP.

**Comment by JamesDeFabia** [comment]
Also the PDF linked is NOT the same as these source files,

**Comment by g-pan** [comment]
Updated and Rebased. @JamesDeFabia please review

File: docs/EN_US/HPCCDataHandling/DH-Mods/DH-Mod1.xml [hunk 1]
```diff
@@ -166,33 +165,24 @@
                   </tbody>
                 </tgroup>
               </informaltable></para>
+
+            <para>Ensure you are using the secure protocol, SFTP, connected to
+            your HPCC Cluster's IP address through port 22 in your SCP tool.
```
Comment (JamesDeFabia) [inline docs/EN_US/HPCCDataHandling/DH-Mods/DH-Mod1.xml]: your Landing Zone's IP or hostname

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
almost there.. just one minor change inline

**Review by JamesDeFabia — APPROVED** [review]
Looks good

**Pull request merged**